core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/ui/utils/TimeUtilities.kt

```diff
@@ -10,8 +10,27 @@ object TimeUtilities {
     @SuppressLint("DefaultLocale")
     @JvmStatic
     fun formatTime(timeInSeconds: Float): String {
-        val minutes = (timeInSeconds / 60).toInt()
-        val seconds = (timeInSeconds % 60).toInt()
-        return String.format("%d:%02d", minutes, seconds)
+        var timeString =""
+        
+        var hours = timeInSeconds / 3600
+        var minutes = (timeInSeconds % 3600) / 60
+        var seconds = timeInSeconds % 60
+       
+        
+         //formatting hours minutes and seconds in string
+        if(hours==0L && minutes==0L){
```

Comment:
this series of if else statement looks a bit convoluted.

Reply:
I don't know how it becomes confusing for you the if else statement. It is straight forward I guess. Also, you can use a Kotlin version when key word as well if you want to avoid long lines of codes.